src/main.rs

```diff
@@ -82,6 +85,65 @@ fn main() {
             }
             Ok(())
         });
+        std::process::exit(0);
+    } else if config.feature_flag_enabled(FeatureFlag::HELP) {
```

Comment:
Could you add comments inside these blocks mentioning what command the user would have run to end up inside them?

The difference between `config.feature_flag_enabled(FeatureFlag::HELP)` vs `args.unstable_options.unstable_help` is not immediately obvious to me right now.

Reply:
I annotated the code paths, but it also makes more sense at this point to just have one `-Z help` option for both feature flags and unstable options.